client/my-sites/checkout/checkout/prepurchase-notices/index.jsx

```diff
@@ -77,6 +85,13 @@ const PrePurchaseNotices = () => {
 		isJetpackMinimumVersion( state, siteId, BACKUP_MINIMUM_JETPACK_VERSION )
 	);
 
+	const scanSlugInCart = cartItemSlugs.find( isJetpackScanSlug );
+	const multisitesIncompatibleProducts = useSelector( ( state ) => {
+		return isJetpackSiteMultiSite( state, siteId )
+			? [ backupSlugInCart, scanSlugInCart ].filter( ( item ) => item ).map( getProductFromSlug )
```

Comment:
I've recently learned that `compact` from `lodash` can filter falsy values as well :)